languages/common-lisp/exercises/concept/sameness/sameness.lisp

```diff
@@ -1,34 +1,33 @@
 (defpackage sameness
   (:use :cl)
   (:export
```

Comment:
If we stick with the function approach, then maybe we can get a bit hacky and put this in the testing file:
```lisp
(do-symbols (sym :sameness)
  (import sym))
```
So we don't have to export everything from this file.

Alternatively, this is less hacky and works in either this stub or the testing file (I think):
```lisp
(do-symbols (sym)
  (when (and (eql (symbol-package sym) (find-package :sameness))
             (fboundp sym))
    (export sym)))
```

Reply:
I lean toward leaving in the list of exported symbols in `defpackage` to keep any fancy stuff out of either the solution or test files. I don't want to have anything in them that might distract/confuse unless it is really a benefit.

Reply:
I think that's probably a good idea, especially for this early exercise :smile:

Reply:
> I lean toward leaving in the list of exported symbols in defpackage to keep any fancy stuff out of either the solution or test files. I don't want to have anything in them that might distract/confuse unless it is really a benefit.

Agreed! Keeping the test files themselves simple is great for the students that use the CLI. In general, the test files should only use language features that you can expect the student to be familiar with (based on the exercise's prerequisites).